src/main/java/com/chnu/model/Transport.java

```diff
@@ -0,0 +1,70 @@
+package com.chnu.model;
+
+import javax.persistence.*;
+import java.io.Serializable;
+
+@Entity
+@Table(name = "transports")
+public class Transport implements Serializable {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "transport_id")
+    private Long transportId;
+
+    @Column(name = "description")
+    private String description;
+
+    @Column(name = "min_weight", nullable = false)
+    private float min_weight;
```

Comment:
wrong field naming convention (min_weight -> minWeight)
apply for other fields, getters and setters

Reply:
do not apply for column names